src/test/resources/regressions/features/builtin/copy-fail3.gobra

```diff
@@ -0,0 +1,35 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+//:: IgnoreFile(/gobra/issue/439/)
```

Comment:
I would create an actual issue for that and then use that number instead of referencing the PR as an issue